### ISSUE TITLE
Line plot should not include markers by default

### ISSUE THREAD

**Issue by mohawkjohn** [Arafatk/magick-rubyplot#18]

**Comment by Arafatk** [comment]
I am assuming you wanted something like this 
![image](https://user-images.githubusercontent.com/11155207/44190563-faa89380-a144-11e8-80a9-87c36ff4debc.png)

So I added it in the [latest commit](https://github.com/Arafatk/magick-rubyplot/commit/53a2b2bd79c6627979227f2d4b05a6d24a1e9e60).   
It looks a bit cleaner now.

**Comment by translunar** [comment]
No. Go try plotting something in matplotlib. This idea of having markers at each data point is an Excel thing, and is not done by default in scientific plotting software. You should have a marker size of 0 by default.

**Comment by arafatkatze** [comment]
Okay, noted.      
Will get back to you on this.

**Comment by arafatkatze** [comment]
@mohawkjohn This change has been introduced as [can be seen in example](https://github.com/Arafatk/magick-rubyplot/blob/master/spec/reference_images/line.png).        
 
What you are saying about marker width is generally very true for example in 
1. Matlab             
![image](https://user-images.githubusercontent.com/11155207/44410142-6e053780-a581-11e8-8fc6-45e0f5ef1486.png)

2. Matplotlib       
![image](https://user-images.githubusercontent.com/11155207/44410166-7c535380-a581-11e8-9689-c3b99cd47512.png)


3. Excel     
![image](https://user-images.githubusercontent.com/11155207/44410217-92f9aa80-a581-11e8-8cc9-8192f14cb64e.png)

Right now I had set the marker size zero. I will adopt all the [options from GrPlot](https://github.com/pgtgrly/GRruby-extension/blob/master/lib/rubyplot/scripting_backends/gr/gr_backend.rb#L41) for my case of line plot in Magick Plot.
